nex/runner.go

```diff
@@ -115,9 +115,19 @@ func RunWorkload(ctx context.Context, logger *slog.Logger) error {
 		argv = strings.Split(RunOpts.Argv, " ")
 	}
 
+	var hsConfig *controlapi.NatsJwtConnectionInfo
+	if RunOpts.HsUrl != "" {
```

Comment:
Is this a fair assumption?  Or should all fields be present?

Reply:
I think using the url is good enough as an indicator for now. We can always make it.more strict later.